apps/meteor/tests/data/livechat/business-hours.ts

```diff
@@ -0,0 +1,11 @@
+import { ILivechatBusinessHour } from '@rocket.chat/core-typings';
+import { credentials, methodCall, request } from '../api-data';
+
+export const saveBusinessHour = async (businessHour: ILivechatBusinessHour) => {
+	await request
+		.post(methodCall('livechat:saveBusinessHour'))
+		.set(credentials)
+		.send({ message: JSON.stringify({ params: [businessHour], msg: 'method', method: 'livechat:saveBusinessHour', id: '101' }) })
+		.expect(200);
```

Comment:
```suggestion
		.expect(200);
		.expect((res: Response) => {
			expect(res.body).to.have.property('success', true);
			const parsedBody = JSON.parse(res.body.message);
			expect(parsedBody).to.have.property('error');
			expect(parsedBody.error).to.have.property('error', 'error-not-allowed');
		});
```

Here's a sample snippet from my other PR where I was parsing the error message from a meteor method. Hope it helps